core/services/chainlink/application.go

```diff
@@ -206,6 +207,13 @@ func NewApplication(opts ApplicationOpts) (Application, error) {
 		opts.CapabilitiesRegistry = capabilities.NewRegistry(globalLogger)
 	}
 
+	// Use a recurring trigger with mock data for testing purposes
```

Comment:
Testing where? Can it be under some flag? It doesn't seem right to always add it.

Reply:
I'll move this inside the `mock_trigger.go`, that way it's only used if the capability is used